app/javascript/map1.js

```diff
@@ -1,10 +1,14 @@
-function initMap() {
-  var map = L.map('map').setView([49.44827, 32.07474], 13);
-  var marker = L.marker([49.44827, 32.07474]).addTo(map);
-  var popup = L.popup()
-    .setLatLng([49.45827, 32.07474])
-    .setContent("We're here!")
-    .openOn(map);
+function initMap1() {
```

Comment:
@lime666 why are you adding this file? as I see we have this logic in `app/javascript/map.js`